### PR TITLE
:heart_eyes: add new attribute for compute change

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by ilmir-k — APPROVED** [review]

File: pos_restaurant_base/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,9 @@
+`2.0.3`
+-------
+
+- New: Attribute ``unit`` for compute change
```
Comment (yelizariev) [inline pos_restaurant_base/doc/changelog.rst]: It's not clear what is for

**Comment by yelizariev** [comment]
It's new feature rather than improvement. Fix commit tag

**Comment by ilmir-k** [comment]
@yelizariev 
waiting for your review

**Review by yelizariev — APPROVED** [review]
It's a new feature, rather than fix

**Pull request merged**